distributedlog-core/src/main/java/org/apache/distributedlog/BKAsyncLogWriter.java

```diff
@@ -206,7 +197,7 @@ private BKLogSegmentWriter getCachedLogSegmentWriter() throws WriteException {
         }
     }
 
-    private Future<BKLogSegmentWriter> getLogSegmentWriter(long firstTxid,
+    private CompletableFuture<BKLogSegmentWriter> getLogSegmentWriter(long firstTxid,
                                                            boolean bestEffort,
```

Comment:
Please also do the code alignment change in this file, at line 201, 212, 242, 272,